Makefile

```diff
@@ -54,3 +55,17 @@ ut-circle: calico_mesos_builder.created dist/calico_isolator
 	--with-xunit --xunit-file=/circle_output/output.xml; RC=$$?;\
 	[[ ! -z "$$COVERALLS_REPO_TOKEN" ]] && coveralls || true; exit $$RC'
 
+builder-rpm:
```

Comment:
Can we rename this to just `rpm`? I know it creates the docker build image, but its end goal is to `make rpm`s